### PR TITLE
Fix Template Import and Export logging issue

### PR DESCRIPTION
**Pull request by jyejare**
Fix following tests failing:
- [x]  test_positive_import_log_to_production
- [x] test_positive_export_log_to_production

### CODE REVIEW

**Comment by jyejare** [comment]
@san7ket , @ntkathole :
```
# py.test tests/foreman/api/test_template.py -k test_positive_import_log_to_production
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15, pytest-3.6.1, py-1.5.4, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 43 items                                                                                                                                                                                               2018-09-25 17:41:33 - conftest - DEBUG - BZ deselect is disabled in settings

collected 43 items / 42 deselected                                                                                                                                                                                

tests/foreman/api/test_template.py .                                                                                                                                                                        [100%]
============================================================================== 1 passed, 42 deselected in 22.31 seconds ===============================================================================


# py.test tests/foreman/api/test_template.py -k test_positive_export_log_to_production
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15, pytest-3.6.1, py-1.5.4, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 43 items                                                                                                                                                                                               2018-09-25 17:42:38 - conftest - DEBUG - BZ deselect is disabled in settings

collected 43 items / 42 deselected                                                                                                                                                                                

tests/foreman/api/test_template.py .                                                                                                                                                                        [100%]

============================================================================== 1 passed, 42 deselected in 45.73 seconds ===============================================================================
```

File: tests/foreman/api/test_template.py [hunk 1]
```diff
@@ -1405,7 +1405,7 @@ def test_positive_import_log_to_production(self):
         self.assertIn(
             'Started POST "/api/v2/templates/import"',
             ssh.command(
-                'tail -10 /var/log/foreman/production.log | grep import'
+                'tail -100 /var/log/foreman/production.log | grep import'
```
Comment (pondrejk) [inline tests/foreman/api/test_template.py]: maybe the bulletproof way would be to grep from everything after a timestamp of when the template started, but if you're sure enough 100 is safe enough buffer, than ack

Comment (jyejare) [reply tests/foreman/api/test_template.py]: Yeah, That would be the great way as well, I will test that and update the PR if it works.

**Comment by jyejare** [comment]
@pondrejk , I changed the code accordingly. Please look again.

**Comment by jyejare** [comment]
@pondrejk @ntkathole @san7ket 
Results post code change :
```
# py.test tests/foreman/api/test_template.py -k test_positive_import_log_to_production
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15, pytest-3.6.1, py-1.5.4, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 43 items                                                                                                                                                                                               2018-10-04 14:59:59 - conftest - DEBUG - BZ deselect is disabled in settings

collected 43 items / 42 deselected                                                                                                                                                                                

tests/foreman/api/test_template.py .                                                                                                                                                                        
============================================================================== 1 passed, 42 deselected in 28.86 seconds ===============================================================================

# py.test tests/foreman/api/test_template.py -k test_positive_export_log_to_production
=============================================================================================== test session starts ===============================================================================================
platform linux2 -- Python 2.7.15, pytest-3.6.1, py-1.5.4, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jitendrayejare/Desktop/RedHat/RoboTelloNew/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.5, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collecting 43 items                                                                                                                                                                                               2018-10-04 15:00:34 - conftest - DEBUG - BZ deselect is disabled in settings

collected 43 items / 42 deselected                                                                                                                                                                                

tests/foreman/api/test_template.py .                                                                                                                                                                        
============================================================================== 1 passed, 42 deselected in 50.44 seconds ===============================================================================
```

**Pull request merged**